Games/GiftOfTheSanctum/Source/GiftOfTheSanctumGame.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include <Zmey/EngineLoop.h>
+#include <Zmey/Game.h>
+#include <Zmey/Utilities.h>
+#include <Zmey/EntityManager.h>
+#include <Zmey/Math/Math.h>
+
+using EntityVector = std::vector<Zmey::EntityId>;
```

Comment:
Zmey::stl::vector, as this is not using the right allocator

Reply:
added todo